clients/extension/scripts/popup.js

```diff
@@ -150,92 +59,61 @@ function createOptionItem(text, id, isChecked) {
-    window.close();
-  } catch (err) {
-    let message = err.friendlyMessage || err.message;
-    let includeReportLink = true;
-
-    // Check for errors in how the user ran Lighthouse and replace with a more
-    // helpful message (and remove 'Report Error' link).
-    for (const [test, replacement] of Object.entries(NON_BUG_ERROR_MESSAGES)) {
-      if (err.message.includes(test) ||
-          (err.friendlyMessage && err.friendlyMessage.includes(test))) {
-        message = replacement;
-        includeReportLink = false;
-        break;
-      }
-    }
-
-    feedbackEl.textContent = message;
-
-    if (includeReportLink) {
-      feedbackEl.className = 'feedback feedback-error';
-      feedbackEl.appendChild(buildErrorCopyButton(message, err));
-    }
-
-    hideRunningSubpage();
-    background.console.error(err);
+function onGenerateReportButtonClick(siteURL, settings) {
+  const url = new URL(`${VIEWER_ORIGIN}${VIEWER_PATH}`);
+  url.searchParams.append('url', siteURL);
+  url.searchParams.append('strategy', settings.device);
+  for (const category of settings.selectedCategories) {
+    url.searchParams.append('category', category);
   }
-
-  isRunning = false;
+  url.searchParams.append('utm_source', 'lh-chrome-ext');
+  window.open(url.href);
 }
 
 /**
  * Generates a document fragment containing a list of checkboxes and labels
  * for the categories.
  * @param {BackgroundPage} background Reference to the extension's background page.
- * @param {Array<string>} selectedCategories
+ * @param {Settings} settings
  */
-function generateOptionsList(background, selectedCategories) {
+function generateOptionsList(background, settings) {
   const frag = document.createDocumentFragment();
 
   background.getDefaultCategories().forEach(category => {
-    const isChecked = selectedCategories.includes(category.id);
+    const isChecked = settings.selectedCategories.includes(category.id);
     frag.appendChild(createOptionItem(category.title, category.id, isChecked));
   });
 
-  const optionsList = find('.options__list');
-  optionsList.appendChild(frag);
+  const optionsCategoriesList = find('.options__categories');
+  optionsCategoriesList.appendChild(frag);
+}
+
+function fillDevToolsShortcut() {
+  const el = find('.devtools-shortcut');
+  const isMac = /mac/i.test(navigator.platform);
+  el.innerText = isMac ? '⌘⌥I (Cmd + Opt + I)' : 'F12';
 }
 
 /**
  * Initializes the popup's state and UI elements.
  */
 async function initPopup() {
+  fillDevToolsShortcut();
+
   chrome.tabs.query({active: true, lastFocusedWindow: true}, function(tabs) {
     if (tabs.length === 0) {
       return;
     }
 
-    siteURL = new URL(tabs[0].url || '');
-
-    // Show the user what URL is going to be tested.
-    find('header h2').textContent = siteURL.origin;
+    siteURL = tabs[0].url || null;
+    const url = siteURL ? new URL(siteURL) : null;
+    const host = url ? url.host : '';
+    if (host.startsWith('localhost')) {
```

Comment:
should have a reason it's disabled, and probably need a place for more of these (e.g. chrome pages, etc)

are we going to warn people that if they're logged in it's not going to work on psi?

Reply:
Would like  to add the limitations into the messaging for Chrome DevTools. (instead of "you can use ..." -> "if ... you should use ...")